web/eXoResources/src/main/webapp/skin/DefaultSkin/portal/webui/component/customization/UIContainerList/Stylesheet.css

```diff
@@ -29,53 +29,3 @@
 	height: 40px; 
 	line-height: 20px;
 }
-

```

Comment:
I think that the entire file isn't used, should be completely deleted isn't it ?

Reply:
fixed